plugins/labhub.py

```diff
@@ -90,12 +111,27 @@ def callback_message(self, msg):
         """Invite the user whose message includes the holy 'hello world'"""
         if not hasattr(self, 'TEAMS'):
             self.TEAMS = LabHub.get_teams(os.environ.get('GH_TOKEN'),
-                                          self.ORG_USERNAME)
+                                          self.GH_ORG_NAME)
```

Comment:
These are unrelated changes to the commit message? Split it into two. One with the constants change and one with the create issue feature added.